ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,354 @@
+        avatarFallback: org.avatarFallback,
+        imageUrl: null,
+      },
+    })),
+  ];
+
+  let state = TransferState.Preparation;
+
+  const goToConfirmation = () => {
+    isTransferringFromUser = $payerStore === identity.metadata.handle;
+    const amountInMicroRad = currency.radToMicroRad($amountStore);
+
+    tx = {
+      fee: transactionFee,
+      messages: [
+        isTransferringFromUser
+          ? {
+              type: MessageType.Transfer,
+              amount: amountInMicroRad,
+              recipient: $recipientStore,
+            }
+          : {
+              type: MessageType.TransferFromOrg,
+              amount: amountInMicroRad,
+              recipient: $recipientStore,
+              orgId: $payerStore,
+            },
+      ],
+    };
+    summary = costSummary(tx);
+    state = TransferState.Confirmation;
+  };
+
+  const backToPreperation = () => {
+    state = TransferState.Preparation;
+  };
+
+  const onConfirmed = async () => {
+    try {
+      await transfer(
+        isTransferringFromUser,
+        $payerStore,
+        parseInt($amountStore),
+        $recipientStore,
+        transactionFee
+      );
+    } catch (error) {
+      notification.error(`Could not transfer funds: ${error.message}`);
+    } finally {
+      pop();
+    }
+  };
+
+  $: amountValidation = amountValidationStore(
+    transactionFee,
+    payerAccountIds[$payerStore]
+  );
+  $: {
+    if ($amountStore && $amountStore.length > 0) validatingAmount = true;
+    if (validatingAmount) amountValidation.validate($amountStore);
+  }
+  $: {
+    if ($recipientStore && $recipientStore.length > 0)
+      validatingRecipient = true;
+    if (validatingRecipient) recipientValidation.validate($recipientStore);
+  }
+  $: disableSubmit =
+    $amountValidation.status !== ValidationStatus.Success ||
+    $recipientValidation.status !== ValidationStatus.Success ||
+    $payerStore.length === 0;
+</script>
+
+<style>
+  .wrapper {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    margin: 92px 0 32px 0;
+    width: 540px;
+  }
+  header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    padding: 2rem;
+    margin-bottom: 1.5rem;
+    background-color: var(--color-foreground-level-1);
+    border: 1px solid var(--color-foreground-level-2);
+    border-radius: 0.25rem;
+  }
+  .icon {
+    height: 2.5rem;
+    width: 2.5rem;
+    border-radius: 1.25rem;
+    background-color: var(--color-primary-level-1);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 1rem;
+  }
+
+  .submit {
+    display: flex;
+    justify-content: flex-end;
+    padding-top: 1.5rem;
+  }
+
+  .from-to {
+    display: grid;
+    grid-template-columns: 13rem 1.5rem 13rem;
+    grid-column-gap: 1rem;
+    margin-top: 1rem;
+  }
+
+  .from {
+    display: flex;
+    justify-content: flex-end;
+  }
+</style>
+
+<ModalLayout dataCy="page">
```

Comment:
This component is feeling reeally big, it might be nice to break it up into some smaller reusable components.

Reply:
Indeed. How do you feel about accepting this as a technical debt of this big feature for which we create a follow-up issue?

Reply:
Makes sense!

Reply:
+1

Reply:
@sarahscott I think @xla is looking at this already in his layout PR's